js/libs/builder.js

```diff
@@ -180,11 +180,13 @@ var app = new Vue({
       }
     },
     deleteField: function(index) {
-      var confirmDelete = confirm("Are you sure you want to delete field?");
-      if (confirmDelete) {
-        this.fields.splice(index, 1);
-        this.activeFieldConfigType = null;
-      }
+      $vm = this;
+      Fliplet.Modal.confirm({ message: 'Are you sure you want to delete field?'}).then(function (result) {
```

Comment:
Use the following configuration for the confirm modal.

```js
Fliplet.Modal.confirm({
  message: '<p>Are you sure you want to delete the following field?</p><p><strong>' + fieldLabel + '</strong></p>',
  buttons: {
    confirm: {
      label: 'Delete field',
      className: 'btn-danger'
    }
  },
  size: 'small'
})
```

Example:

![image](https://user-images.githubusercontent.com/290733/64629215-b260b100-d3ea-11e9-9ab1-b3b9001a2fc1.png)

Reply:
@tonytlwu Fixed